### DISCUSSION TITLE
The filament sensor not working.

### DISCUSSION THREAD

**Discussion by kickfliph** [mriscoc/Ender3V2S1#1363] [Q&A]
Hello there,

I have been using your firmware since I have had this printer E3S1. So far has been fantastic but, one day I started to print and I didn't realize that I was running short on filament so when I came back the print was already lost to my surprise the filament sensor didn't stop the print, why? So I did my mechanical check looking for a loosened or broken cable, faulty connection, or anything that could lead me to the reason why it's not working. I can see the blue LED goes on when I fish the filament and off there is none inserted so this brings me to the question if this can be something related to the firmware.

Regards,

**Comment by mriscoc** [comment]
The filament sensor is disabled by default and must be configured according with your hardware:
https://github.com/mriscoc/Ender3V2S1/wiki/Filament-Settings-Menu

**Reply by mriscoc** [reply]
https://github.com/mriscoc/Ender3V2S1/discussions/1367#discussioncomment-11087968
